OrbitGl/TimeGraph.h

```diff
@@ -67,23 +67,30 @@ class TimeGraph {
     kPartlyVisible,
     kFullyVisible,
   };
-  void HorizontallyMoveIntoView(VisibilityType vis_type, TickType min, TickType max, double distance = 0.3); 
-  void HorizontallyMoveIntoView(VisibilityType vis_type, const TextBox* text_box, double distance = 0.3);
+  void HorizontallyMoveIntoView(VisibilityType vis_type, TickType min,
+                                TickType max, double distance = 0.3);
+  void HorizontallyMoveIntoView(VisibilityType vis_type,
+                                const TextBox* text_box, double distance = 0.3);
   void VerticallyMoveIntoView(const TextBox* text_box);
   double GetTime(double a_Ratio);
   double GetTimeIntervalMicro(double a_Ratio);
   void Select(const TextBox* a_TextBox);
   enum class JumpScope {
     kGlobal,
+    kSameDepth,
     kSameThread,
```

Comment:
From what I see kSameThread is not used anywhere, isn't it?

Reply:
If you move On and Down (arrow keys) you are moving in the same thread (at least by now).
If you move Left and Right you are moving in the same thread but also in the same depth (for example saying that we are jumping to the closer previous event in the same thread is false, because we are preserving the Depth). Do you have some suggestion for better names?
Maybe I have to split the jumpings and the usual-arrow-keys-move.

Reply:
Ah sorry. No it's fine. My bad. I only scrolled up, but not down. Now I see the usages of kSameThread. Thanks!